nemo/collections/nlp/modules/common/megatron/megatron_bert.py

```diff
@@ -137,7 +137,7 @@ def restore_weights(self, restore_path: str):
         self._restore_path = restore_path
         if os.path.isfile(restore_path):
             logging.info(f'restore_path: {restore_path} is a file. Assuming no megatron model parallelism')
-            state_dict = torch.load(restore_path)
+            state_dict = torch.load(restore_path, map_location=lambda storage, loc: storage)
```

Comment:
Could you change this to
```
state_dict = torch.load(restore_path, map_location='cpu')
```
This is how megatron-lm has it: https://github.com/NVIDIA/Megatron-LM/blob/79888e162425e8d64043a9597ee14751bd4b53d1/megatron/checkpointing.py#L187

Reply:
Thanks! Done